.github/workflows/docker-build.yml

```
@@ -15,61 +15,146 @@ on:
 
       - name: Login to Container registry
         uses: docker/login-action@v3
+        if: github.actor != 'dependabot[bot]'
         with:
           registry: ghcr.io
-          username: ${{ github.actor }}
+          username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Set APP_VERSION env
         run: echo APP_VERSION=$(echo ${GITHUB_REF} | rev | cut -d'/' -f 1 | rev ) >> ${GITHUB_ENV}
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:18: Quote this to prevent word splitting

(shellcheck)